test/cli/dump-config/expected-stdout

```diff
@@ -24,4 +24,4 @@ Configuration: {
   "cpuProfile": ""
 }
 Slicing table "mytable".
-Table "mytable" sliced: in/out: 1 / 1 slices, 22B / 10B bytes, 3 rows, manifest created.
\ No newline at end of file
+Table "mytable" sliced: in/out: 1 / 1 slices, 22B / 42B bytes, 3 rows, manifest created.
```

Comment:
Prejavilo sa to aj v existujucom E2E teste (v inych je wildcard `*B`).

Sedi to s tym co reportuje OS:
![image](https://github.com/keboola/processor-split-table/assets/19371734/92a66834-761e-4842-b6ae-cf2e25520719)

Reply:
(Ze je vystupna velkost vacsia ako vstupna je normalne, ked sa jedna o velmi maly text, ktory sa zagzippuje).